api/api.py

```diff
@@ -9,99 +9,204 @@
 
 from . import core
 from . import jobs
-from . import users
-from . import projects
-from . import sessions
-from . import acquisitions
-from . import collections
+from handlers import listhandler
+from handlers import userhandler
+from handlers import grouphandler
+from handlers import containerhandler
+from handlers import collectionshandler
 
+#regexes used in routing table:
+routing_regexes = {
+    'group_id_re': '[0-9a-z][0-9a-z.@_-]{0,30}[0-9a-z]',
+    'cid_re': '[0-9a-f]{24}',
```

Comment:
Every single one of these regexes needs a comment explaining:

1. What the regex is for
1. How the regex is put together & how it operates

The group ID regex is a great example of a regex that needs verbose justification.